prisma/schema.prisma

```diff
@@ -38,7 +38,7 @@ model User {
 model Profile {
   id             Int              @id @default(autoincrement())
   user           User             @relation(fields: [userId], references: [id])
-  userId         Int              @unique
+  userId         Int              
```

Comment:
since the profile is one to one, why wouldn't the userId be unique?

Reply:
I removed it when troubleshooting, yeah it should still be unique

Reply:
defo should be unique - enforce the constraint, don't just assume it ;)